src/objects/Player.gd

```diff
@@ -25,8 +25,28 @@ func _unhandled_input(event: InputEvent) -> void:
 		return
 	if event is InputEventMouseMotion:
 		var mm_event := event as InputEventMouseMotion
-		rotation.y = wrapf(rotation.y - mm_event.relative.x * MOUSE_SENS.x, 0, TAU)
-		camera.rotation.x = clamp(camera.rotation.x - mm_event.relative.y * MOUSE_SENS.y, -PI / 2, PI / 2)
+		var relative := mm_event.relative
```

Comment:
worth making this it's own function (maybe `handle_mouse_motion()`)? Or do you want to keep it here

Reply:
Not a bad idea actually

Reply:
Done in 91a82144